astropy/visualization/wcsaxes/tests/test_images.py

```diff
@@ -58,6 +58,16 @@ def test_image_plot(self):
         ax.coords[0].set_ticks([-0.30, 0., 0.20] * u.degree, size=5, width=1)
         return fig
 
+    @pytest.mark.remote_data(source='astropy')
+    @pytest.mark.mpl_image_compare(baseline_dir=IMAGE_REFERENCE_DIR,
+                                   tolerance=0, style={})
+    def test_axes_off(self):
+        # Test for turning the axes off
+        fig = plt.figure(figsize=(6, 6))
+        ax = fig.add_axes([0.1, 0.1, 0.8, 0.8], projection=WCS(self.msx_header))
+        ax.set_axis_off()
```

Comment:
Can you add plotting of an image to make sure there is something in the figure and clipped to the axes?